scripts/install_grpc

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+# This script is only intended to run on the CI machines. Not for local
+# development.
+
+if [ "" == "$1" ] || [ "" == "$2" ] || [ "" == "$3" ] ; then
```

Comment:
Why not use `$#` (i.e. `[ "$#" -ne 3]`)?

Reply:
That doens't hurt, I guess, but $# doesn't prevent against empty parameters, which can happen quite easily if the script with unset variables passed as parameters.  Added the $# check.

Reply:
> but $# doesn't prevent against empty parameters, which can happen quite easily if the script with unset variables passed as parameters.

Ah! I see. Makes sense (wasn't sure why we would have empty strings as inputs - but I see now)